src/version-github.ts

```diff
@@ -18,7 +17,7 @@ export default class GitHubVersioner extends BaseVersioner {
     super();
     this.owner = opts.owner;
     this.repo = opts.repo;
-    this.gitHub = new Octokit({ auth: process.env.GITHUB_TOKEN ?? undefined });
+    this.gitHub = new Octokit({ auth: opts.authOptions });
```

Comment:
Should we preserve the old behavior? i.e.
```suggestion
    this.gitHub = new Octokit({ auth: opts.authOptions ?? process.env.GITHUB_TOKEN });
```

Reply:
I don't think anyone is using this yet and I prefer the shape of the newer option without the hidden fallback, so I changed it like this.